openeo_udp/biopar/biopar.json

```diff
@@ -0,0 +1,174 @@
+    "id": "biopar",
+    "links": [
+        {
+            "href": "https://raw.githubusercontent.com/ESA-APEx/apex_algorithms/refs/heads/biopar_vito/openeo_udp/biopar/biopar.json",
```

Comment:
This URL will also break once the feature branch biopar_vito is removed

Reply:
change it to
https://raw.githubusercontent.com/ESA-APEx/apex_algorithms/3b5a011a90f4a3050ff8fdf69ca5bc2fd1535881/openeo_udp/biopar/biopar.json